packaging/docker/welcome.sh

```
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+ULTIMATE_CONFIG_PATH="/home/ultimate/config"
+
+check_and_print_version() {
+    for cmd in "${@}"; do
+        if command -v "${cmd}" >/dev/null 2>&1; then
+            "${cmd}" --version
+            return
+        fi
```

else?

---

The `else` branch was intentionally omitted because the script is very general and intended to print version information for each Ultimate product. Therefore, the script checks all known launcher names and prints the version information for the first one that exists in the Docker container.

---

Ah, that makes sense -- I was afraid that this leads to silent failures when no command matches